plyflatten/utils.py

```diff
@@ -9,25 +9,39 @@ class InvalidPlyCommentsError(Exception):
     pass
 
 
-def utm_proj(utm_zone):
+def crs_proj(utm_zone, crs_type="UTM"):
     """
     Return a pyproj.Proj object that corresponds
-    to the given utm_zone string
+    to the given utm_zone string or EPSG code
 
     Args:
-        utm_zone (str): UTM zone number + hemisphere (eg: '30N')
+        crs_type (str): 'UTM' (default) or 'EPSG'
+        utm_zone (str): UTM zone number + hemisphere (eg: '30N') or EPSG code
 
     Returns:
         pyproj.Proj: object that can be used to transform coordinates
     """
-    zone_number = utm_zone[:-1]
-    hemisphere = utm_zone[-1]
-    return pyproj.Proj(
-        proj="utm", zone=zone_number, ellps="WGS84", datum="WGS84", south=(hemisphere == "S")
-    )
+    if crs_type == "UTM":
+        zone_number = utm_zone[:-1]
+        hemisphere = utm_zone[-1]
+        return pyproj.Proj(
+            proj="utm", zone=zone_number, ellps="WGS84", datum="WGS84", south=(hemisphere == "S")
+        )
+    elif crs_type == "EPSG":
+        return pyproj.Proj("epsg:{}".format(utm_zone))
```

Comment:
the variable name `utm_zone` is a bit misleading as it actually contains either a UTM zone or a EPSG code. Shouldn't we rename it to something more generic, like `crs` for example?